cma/purecma.py

```diff
@@ -674,7 +674,7 @@ def legend_(*args, **kwargs):
         title_('Coordinate-wise STDs w/o sigma')
         grid(True)
         xlabel('iterations' + strpopsize)
-        _stdout.flush()
```

Comment:
it wasn't a typo, was it?

Reply:
Heh. No. It's that _stdout is supposed to be the original one, not whatever stdout has been renamed to (print will go to the latter).